frontend/src/components/maps/MineMapLeaflet.js

```diff
@@ -213,6 +230,67 @@ class MineMapLeaflet extends Component {
     });
   };
 
+  addLatLongCircle = () => {
+    if (this.props.lat && this.props.long && !this.props.mineName) {
+      L.circle([this.props.lat, this.props.long], {
+        color: "red",
+        fillColor: "#f03",
+        fillOpacity: 0.25,
+        radius: 500,
+        stroke: false,
+      }).addTo(this.map);
+    }
+  };
+
+  addMinePinClusters = async () => {
+    // Add Clustered MinePins
+    this.markerClusterGroup = L.markerClusterGroup({ animate: false });
+    this.props.minesBasicInfo.map(this.createPin);
+    this.map.addLayer(this.markerClusterGroup);
+    this.addLatLongCircle();
+  };
+
+  addWidgets = async () => {
```

Comment:
Same question here. Perhaps it's a misunderstanding around `async` functions? They're still invoked synchronously. And if all of the statements in the function are also synchronous, there is no performance benefit provided.

Ex.
```javascript
const fn = async () => {
  console.log("async function");
}

fn();
console.log("sync");
```

Output:
```
async function
sync
```